Sources/InternxtSwiftCore/Types/DriveTypes.swift

```diff
@@ -119,3 +119,50 @@ public struct GetFolderMetaByIdResponse: Decodable {
 public struct GetFileMetaByIdResponse: Decodable {
    
 }
+
+
+public struct CreateFileData: Encodable {
+    public let fileId: String
+    public let type: String?
+    public let bucket: String
+    public let size: Int
+    public let folder_id: Int
```

Comment:
Transform this when receiving the request, so you are not coupled

Reply:
this is the payload to SEND the request, backend expects a parameter called folder_id https://github.com/internxt/sdk/blob/master/src/drive/storage/index.ts#L221

- Encodable protocol allow to encode the struct to a JSON
- Decodable protocol allow to decode the JSON to an struct (with validation, which means if we receive an String instead of an Int where we defined an Int, the decoding will fail)